File: java/src/test/java/com/libutil/test/file/DecodeBase64FileToFileTest.java
package com.libutil.test.file;

import java.io.IOException;

import com.libutil.FileUtil;
import com.libutil.test.Log;

public class DecodeBase64FileToFileTest {

  public static void main(String args[]) {
    try {
      test();
    } catch (IOException ioe) {
      Log.e(ioe);
    }
  }

  private static void test() throws IOException {
    FileUtil.decodeBase64FileToFile("C:/test/b64image.txt", "C:/tmp/image.jpg");
    FileUtil.decodeBase64FileToFile("C:/test/NotFound.txt", "C:/tmp/notfound.txt");
    Log.d("OK");
  }

}

File: python/test/base64/decode_base64_file_to_file_test.py
import os
import sys

sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
import util

def main():
  util.decode_base64_file_to_file('C:/test/b64image.txt', 'C:/tmp/image.jpg')
  print('OK')

main()
